pages/projects.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import { Navbar, Footer, Breadcrum } from "../components";
+
+function Projects() {
+  return (
+    <>
+      <Navbar />
+
+      <main className="bg-[#3331C4] pb-12">
+        {/* Change this to the current Active Page */}
+        <Breadcrum currentPage="Sample" />
```

Comment:
you forgot to update the breadcrumbs prop

Reply:
Done

Reply:
I forgot to make some commits but I will do them later